src/logging.h

```diff
@@ -169,12 +203,7 @@ BCLog::Logger& LogInstance();
 /** Return true if log accepts specified category, at the specified level. */
 static inline bool LogAcceptCategory(BCLog::LogFlags category, BCLog::Level level)
 {
-    // Log messages at Warning and Error level unconditionally, so that
-    // important troubleshooting information doesn't get lost.
-    if (level >= BCLog::Level::Warning) {
-        return true;
-    }
-    return LogInstance().WillLogCategory(category);
+    return LogInstance().WillLogCategoryLevel(category, level);
 }
```

Comment:
`LogAcceptCategory()` is now the same as `WillLogCategoryLevel()` and thus looks unnecessary. It is used in just 6 places. Consider replacing all of those with `LogInstance().WillLogCategoryLevel(category, level)` and removing `LogAcceptCategory()`.

Reply:
I plan to do this in the parent PR in the refactorings to come.